### PR TITLE
gpperfmon: remove filerep stat collection

### PR DESCRIPTION
**Pull request by Chibin**
Signed-off-by: Larry Hamel <lhamel@pivotal.io>

### CODE REVIEW

File: gpAux/gpperfmon/src/gpmon/gpperfmon41.sql [hunk 1]
```diff
@@ -18,59 +18,6 @@
 --   message                    text message associated with this event
 create table public.health_history (ctime timestamp(0) not null, hostname varchar(64) not null, symptom_code int not null, detailed_symptom_code int not null, description text not null, status text not null, message text not null) distributed by (ctime) partition by range (ctime)(start (date '2010-01-01') end (date '2010-02-01') EVERY (interval '1 month'));
 
--- TABLE: filerep_history
```
Comment (danielgustafsson) [inline gpAux/gpperfmon/src/gpmon/gpperfmon41.sql]: For any current installations upgrading to 5, will this just work with filerep stats no longer being populated or is there an upgrade path required?

**Comment by Chibin** [comment]
> For any current installations upgrading to 5, will this just work with filerep stats no longer being populated or is there an upgrade path required?

Good question, I think it'll just be a left over table that no longer gets populated.

Which begs question, do the gucs gets saved during upgrade (I assume yes)? There's a particular guc that enables gpperfmon. And if the gpperfmon guc gets saved as enabled, but gpperfmon hasn't been installed in the gpdb5. Interesting things may happen..

**Comment by ashwinstar** [comment]
Sorry missing the context for the change on why? as neither PR nor commit message mentions it. Like no one uses it, its not relevant, it never worked.

**Comment by Chibin** [comment]
> Sorry missing the context for the change on why? as neither PR nor commit message mentions it. Like no one uses it, its not relevant, it never worked.

I think the consensus is that no one uses it and that it's very finicky with the information filerep history provides, but I'll defer to @skahler-pivotal for more information.

**Comment by skahler-pivotal** [comment]
The implementation was flakey and got very little use, additionally we'll be looking at changing away from filerep in the near future.

**Pull request merged**